Dockerfile

```diff
@@ -26,4 +26,7 @@ RUN if [ "${BUILD_TYPE}" = "dev" ]; then \
     fi \
     && pip install ${PIP_OPTIONS} jupyter notebook
 
+# Install RDB bindings.
+RUN pip install ${PIP_OPTIONS} PyMySQL cryptography psycopg2-binary
```

Comment:
[Question]

Is `cryptography` required?
I know it is installed in `tests-rdbstorage`.

https://github.com/optuna/optuna/blob/e0fd830471cde878e2882c2ba1adaed307edd7c9/.github/workflows/tests-rdbstorage.yml#L89

But tests passed without the library.
https://github.com/himkt/optuna/pull/4/commits/504d24a8e2a3bc508b6d22a62b5c23beedb2a9db

Reply:
Right, it's an optional dependency in pymysql for handling MySQL password (e.g. [setup.py](https://github.com/PyMySQL/PyMySQL/blob/master/setup.py#L21), [usage in code](https://github.com/PyMySQL/PyMySQL/blob/6ccbecc1a0dfd04065b081950d2d35b1dac0aaa8/pymysql/_auth.py#L137-L155)). It's suggested practice in my limited understanding. 

Cc. @toshihikoyanase do you know if we can or should remove it?

Reply:
📝  Asked @toshihikoyanase and @himkt. This library is apparently utilized automatically if installed to encrypt traffic. It seems fine to include it in our images.

Reply:
> This library is apparently utilized automatically if installed to encrypt traffic.

For example, `PyMySQL` uses it internally.
https://github.com/PyMySQL/PyMySQL/blob/6ccbecc1a0dfd04065b081950d2d35b1dac0aaa8/pymysql/_auth.py#L7-L14

Reply:
It by the way seems like users will experience errors in case the dependency is missing (and `sha256_password` is specified).